packages/@lwc/engine-core/src/framework/reporting.ts

```diff
@@ -11,6 +11,8 @@ export const enum ReportingEventId {
     CrossRootAriaInSyntheticShadow = 0,
     CompilerRuntimeVersionMismatch = 1,
     NonStandardAriaReflection = 2,
+    TemplateMutation = 3,
+    StylesheetMutation = 4,
```

Comment:
I didn't need to split these up, but I felt I might as well. I really doubt anybody is doing the `StylesheetMutation` (i.e. `stylesheet.$scoped$ = true`).